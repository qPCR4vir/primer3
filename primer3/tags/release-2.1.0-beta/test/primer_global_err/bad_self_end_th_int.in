SEQUENCE_ID=bad_self_end_th_int
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_MAX_SELF_END_TH=-1
=
