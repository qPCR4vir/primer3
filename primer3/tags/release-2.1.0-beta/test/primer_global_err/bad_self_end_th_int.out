SEQUENCE_ID=bad_self_end_th_int
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_MAX_SELF_END_TH=-1
PRIMER_ERROR=Illegal value for internal oligo complementarity restrictions
=
